.github/dependabot.yml

```diff
@@ -4,3 +4,13 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "composer"
```

Comment:
This is done, so the dependabot doesn't try to upgrade Drupal core to 10 for 1.x branch.
cc: @danepowell / @secretsayan